@types/resolve-promise-cb.d.ts

```diff
@@ -0,0 +1 @@
+export function resolvePromiseCb<T>(err?: Error, res?: T, cb?: (err?: Error, res?: T) => void): void|Promise<T>
```

Comment:
seems this is not included in index.d.ts:
```ts
export { promiseFlat } from './@types/promise-flat'
export { promiseSleep } from './@types/promise-sleep'
export { promiseTimeout, PromiseTimeoutError } from './@types/promise-timeout'
export { resolvePromiseCb } from './@types/resolve-promise-cb'
```

Reply:
fixed